scripts/generate_sitemap/utils.ts

```diff
@@ -0,0 +1,81 @@
+import { concat, chain } from 'lodash';
+import { SitemapItemLoose } from 'sitemap';
+import urlJoin from 'url-join';
+import regions from '../../src/common/regions';
+import { allCaseStudies } from '../../src/cms-content/learn';
+import articles from '../../src/cms-content/articles';
+import shareImagesData from '../../src/assets/data/share_images_url.json';
+
+const shareImageBaseUrl = shareImagesData.share_image_url;
+
+export function getLocationPageItems(): SitemapItemLoose[] {
+  return chain(regions.all())
+    .map(region => ({
+      url: region.relativeUrl,
+    }))
+    .value();
+}
+
+export function getLearnPageItems(): SitemapItemLoose[] {
+  const topLevelRelativeUrls = [
+    '/learn',
+    '/glossary',
+    '/faq',
+    '/case-studies',
+    '/deep-dives',
+    '/covid-risk-levels-metrics',
+  ];
+
+  const caseStudyUrls = allCaseStudies.map(caseStudy =>
+    urlJoin('/case-studies/', caseStudy.caseStudyId),
+  );
+
+  const deepDiveUrls = articles.map(article =>
+    urlJoin('/deep-dives', article.articleID),
+  );
+
+  const allLearnUrls = concat(
+    topLevelRelativeUrls,
+    caseStudyUrls,
+    deepDiveUrls,
+  );
+
+  return allLearnUrls.map(relativeUrl => ({
+    url: relativeUrl,
+  }));
+}
+
+export function getTopLevelPageItems(): SitemapItemLoose[] {
+  const relativeUrlList = [
+    '/',
+    '/donate',
+    '/about',
+    '/tools',
+    '/contact',
+    '/terms',
+    '/privacy',
+  ];
+  return relativeUrlList.map(relativeUrl => ({
+    url: relativeUrl,
+  }));
+}
+
+export function getHomeImageItems(): SitemapItemLoose[] {
+  const homeMapUrl = urlJoin(shareImageBaseUrl, 'home.png');
```

Comment:
Oooh nice!  Though this is the sharing image but I think we probably want to use the one I made to put in our `<meta name='image'>` tag instead: https://github.com/covid-projections/covid-projections/blob/40c34030f374e2da845a79bd4f06bd6721c53e6e/src/common/urls.ts#L48